StudioCore/MsbEditor/ModelEditorScreen.cs

```diff
@@ -293,12 +293,12 @@ public void ReloadAssetBrowser()
 
         public override void Save()
         {
-            
+            TaskManager.UpdateLastActionString("Model Saving is not implemented");
```

Comment:
Again this is kinda introducing more global hardcoded state, so I'd prefer to hold off on this for now. It might be good to have an actual error console down the line.